src/WebJobs.Script/packages.config

```diff
@@ -7,17 +7,19 @@
   <package id="Microsoft.AspNet.WebApi.Core" version="5.2.3" targetFramework="net45" />
   <package id="Microsoft.AspNet.WebHooks.Common" version="1.2.0-rc1" targetFramework="net46" />
   <package id="Microsoft.AspNet.WebHooks.Receivers" version="1.2.0-beta6" targetFramework="net46" />
+  <package id="Microsoft.Azure.ApiHub.Sdk" version="0.2" targetFramework="net46" />
   <package id="Microsoft.Azure.DocumentDB" version="1.5.3" targetFramework="net46" />
   <package id="Microsoft.Azure.Mobile.Client" version="2.0.1" targetFramework="net46" />
   <package id="Microsoft.Azure.NotificationHubs" version="1.0.3" targetFramework="net46" />
   <package id="Microsoft.Azure.ServiceBus.EventProcessorHost" version="1.4.0" targetFramework="net46" />
   <package id="Microsoft.Azure.WebJobs" version="1.1.2-alpha-10286" targetFramework="net46" />
-  <package id="Microsoft.Azure.WebJobs.Core" version="1.1.2-alpha-10286" targetFramework="net46" />
+  <package id="Microsoft.Azure.WebJobs.Core" version="1.1.2-alpha-10286" targetFramework="net46" /> 
   <package id="Microsoft.Azure.WebJobs.Extensions" version="1.0.2-alpha-10266" targetFramework="net46" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.DocumentDB" version="1.0.0-alpha-10266" targetFramework="net46" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.EasyTables" version="1.0.0-alpha-10266" targetFramework="net46" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.NotificationHubs" version="1.0.0-alpha-10266" targetFramework="net46" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.SendGrid" version="1.0.2-alpha-10266" targetFramework="net46" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.ApiHub" version="1.0.0-alpha-10266" targetFramework="net46" />

```

Comment:
Same comment made by @mathewc above.... looks like we're downgrading the packages here.